### PR TITLE
Created constants and optional_inputs and got rid of the confusing expert_inputs name

### PR DESCRIPTION
**Pull request by fletchapin**

### CODE REVIEW

File: aide_design/constants.py [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 Last modified: Fri Jul 7 2017
```
Comment (ethan92429) [inline aide_design/constants.py]: Take off this Sage business and last modified stuff... it's all stored on GitHub.

File: aide_design/constants.py [hunk 2]
```diff
@@ -6,6 +6,9 @@
...
 by: Sage Weber-Shirk
+
+This file contains constants which represent physical properties and scientific
```
Comment (ethan92429) [inline aide_design/constants.py]: Make it [numpy style](https://github.com/numpy/numpy/blob/master/doc/example.py)

File: aide_design/floc_model.py [hunk 3]
```diff
@@ -143,24 +140,27 @@ def sep_dist_aluminum(ConcAluminum):
 
 @u.wraps(1/u.m**3, [u.kg/u.m**3, u.m], False)
 def num_clay(ConcClay, material):
+    """Return the number of clay particles in suspension."""
```
Comment (ethan92429) [inline aide_design/floc_model.py]: numpy style...

File: aide_design/floc_model.py [hunk 4]
```diff
@@ -54,22 +53,20 @@ def define_Precip(self, diameter, density, molecweight, alumMPM):
...
                 'AlOH3', AluminumMPM=2)
-Alum.define_Precip((70 * u.nm).to(u.m).magnitude, 2420, 0.078, 1)
 
+Alum.define_Precip((70 * u.nm).to(u.m).magnitude, 2420, 0.078, 1)
 
 HumicAcid = Chemical('Humic Acid', 72 * 10**-9, 1780, None, 'Humic Acid')
 
 
 ################### Necessary Constants ###################
 # Fractal diameter, based on data from Adachi.
```
Comment (ethan92429) [inline aide_design/floc_model.py]: should be dimension, not diameter. can we more specifically reference Adachi,... see [here](http://online.liebertpub.com/doi/abs/10.1089/ees.2015.0302)

File: aide_design/floc_model.py [hunk 5]
```diff
@@ -143,24 +140,27 @@ def sep_dist_aluminum(ConcAluminum):
...
 def sep_dist_clay(ConcClay, material):
     """Return the separation distance between clay particles."""
-    return ((material.Density / ConcClay) * ((np.pi * material.Diameter**3) / 6))**(1/3)
+    return ((material.Density/ConcClay)*((np.pi
+                            * material.Diameter ** 3)/6))**(1/3)
 
 
 @u.wraps(1/u.m**3, [u.kg/u.m**3, None], False)
 def num_nanoclusters(ConcAluminum, coag):
+    """Return the number of Aluminum nanoclusters."""
```
Comment (ethan92429) [inline aide_design/floc_model.py]: numpy

File: aide_design/floc_model.py [hunk 6]
```diff
@@ -143,24 +140,27 @@ def sep_dist_aluminum(ConcAluminum):
...
     return (ConcAluminum / (dens_alum_nanocluster(coag).magnitude
-                            * np.pi * coag.Diameter**3
-                            ))
+                            * np.pi * coag.Diameter**3))
 
 
 @u.wraps(None, [u.kg/u.m**3, u.kg/u.m**3, None, None], False)
 def frac_vol_floc_initial(ConcAluminum, ConcClay, coag, material):
+    """Return the fraction of flocs initially present."""
```
Comment (ethan92429) [inline aide_design/floc_model.py]: numpy

File: aide_design/floc_model.py [hunk 7]
```diff
@@ -295,56 +291,61 @@ def gamma_humic_acid_to_coag(ConcAl, ConcNatOrgMat, NatOrgMat, coag):
 
 @u.wraps(None, [u.m, u.kg/u.m**3, u.kg/u.m**3, u.kg/u.m**3, None,
                 None, None, u.dimensionless], False)
-def _pacl_term(DiamTube, ConcClay, ConcAl, ConcNatOrgMat, NatOrgMat, 
+def pacl_term(DiamTube, ConcClay, ConcAl, ConcNatOrgMat, NatOrgMat,
                coag, material, RatioHeightDiameter):
+    """"""
```
Comment (ethan92429) [inline aide_design/floc_model.py]: huh?

File: aide_design/floc_model.py [hunk 8]
```diff
@@ -266,26 +266,22 @@ def gamma_coag(ConcClay, ConcAluminum, coag, material,
...
-                       )
-            )
+                          / ratio_clay_sphere(RatioHeightDiameter))
+                       ))
 
 
 @u.wraps(None, [u.kg/u.m**3, u.kg/u.m**3, None, None], False)
 @ut.list_handler
 def gamma_humic_acid_to_coag(ConcAl, ConcNatOrgMat, NatOrgMat, coag):
+    """"""
```
Comment (ethan92429) [inline aide_design/floc_model.py]: huh?

Comment (ethan92429) [reply aide_design/floc_model.py]: The fraction of the coagulant that is coated with humic acid.

File: aide_design/floc_model.py (hunk 7 above)
Comment (ethan92429) [reply aide_design/floc_model.py]: Fraction of the surface area that is covered with coagulant that is not covered with humic acid.

File: aide_design/floc_model.py [hunk 9]
```diff
@@ -295,56 +291,61 @@ def gamma_humic_acid_to_coag(ConcAl, ConcNatOrgMat, NatOrgMat, coag):
...
 
 
-@u.wraps(None, [u.m, u.kg/u.m**3, u.kg/u.m**3, u.kg/u.m**3, 
+@u.wraps(None, [u.m, u.kg/u.m**3, u.kg/u.m**3, u.kg/u.m**3,
                 None, None, None, u.dimensionless], False)
-def alpha_pacl_clay(DiamTube, ConcClay, ConcAl, ConcNatOrgMat, 
+def alpha_pacl_clay(DiamTube, ConcClay, ConcAl, ConcNatOrgMat,
                     NatOrgMat, coag, material, RatioHeightDiameter):
-    PAClTerm = _pacl_term(DiamTube, ConcClay, ConcAl, ConcNatOrgMat,
+    """"""
```
Comment (ethan92429) [inline aide_design/floc_model.py]: Probability of a collision between @monroews

File: aide_design/floc_model.py [hunk 10]
```diff
@@ -295,56 +291,61 @@ def gamma_humic_acid_to_coag(ConcAl, ConcNatOrgMat, NatOrgMat, coag):
...
                                            DiamTube, RatioHeightDiameter)))
 
 
 @u.wraps(None, [u.m, u.kg/u.m**3, u.kg/u.m**3, u.kg/u.m**3,
                 None, None, None, u.dimensionless], False)
-def alpha_pacl_pacl(DiamTube, ConcClay, ConcAl, ConcNatOrgMat, 
+def alpha_pacl_pacl(DiamTube, ConcClay, ConcAl, ConcNatOrgMat,
                     NatOrgMat, coag, material, RatioHeightDiameter):
-    PAClTerm = _pacl_term(DiamTube, ConcClay, ConcAl, ConcNatOrgMat,
+    """"""
```
Comment (ethan92429) [inline aide_design/floc_model.py]: @monroews

File: aide_design/floc_model.py [hunk 11]
```diff
@@ -295,56 +291,61 @@ def gamma_humic_acid_to_coag(ConcAl, ConcNatOrgMat, NatOrgMat, coag):
...
     return PAClTerm ** 2
 
 
 @u.wraps(None, [u.m, u.kg/u.m**3, u.kg/u.m**3, u.kg/u.m**3,
                 None, None, None, u.dimensionless], False)
-def alpha_pacl_nat_org_mat(DiamTube, ConcClay, ConcAl, ConcNatOrgMat, 
+def alpha_pacl_nat_org_mat(DiamTube, ConcClay, ConcAl, ConcNatOrgMat,
                            NatOrgMat, coag, material, RatioHeightDiameter):
-    PAClTerm = _pacl_term(DiamTube, ConcClay, ConcAl, ConcNatOrgMat,
+    """"""
```
Comment (ethan92429) [inline aide_design/floc_model.py]: @monroews

File: aide_design/floc_model.py [hunk 12]
```diff
@@ -295,56 +291,61 @@ def gamma_humic_acid_to_coag(ConcAl, ConcNatOrgMat, NatOrgMat, coag):
...
 
-@u.wraps(None, [u.m, u.kg/u.m**3, u.kg/u.m**3, u.kg/u.m**3, 
+@u.wraps(None, [u.m, u.kg/u.m**3, u.kg/u.m**3, u.kg/u.m**3,
                 None, None, None, u.dimensionless], False)
-def alpha(DiamTube, ConcClay, ConcAl, ConcNatOrgMat, 
+def alpha(DiamTube, ConcClay, ConcAl, ConcNatOrgMat,
           NatOrgMat, coag, material, RatioHeightDiameter):
-    return (alpha_pacl_nat_org_mat(DiamTube, ConcClay, ConcAl, ConcNatOrgMat, 
-                                   NatOrgMat, coag, material, 
+    """"""
```
Comment (ethan92429) [inline aide_design/floc_model.py]: @monroews

File: aide_design/floc_model.py [hunk 13]
```diff
@@ -355,16 +356,15 @@ def alpha(DiamTube, ConcClay, ConcAl, ConcNatOrgMat,
 def pc_viscous(EnergyDis, Temp, Time, DiamTube,
                ConcClay, ConcAl, ConcNatOrgMat, NatOrgMat,
                coag, material, FittingParam, RatioHeightDiameter):
+    """"""
```
Comment (ethan92429) [inline aide_design/floc_model.py]: @monroews

File: aide_design/physchem.py [hunk 14]
```diff
@@ -43,16 +47,16 @@ def diam_circle(AreaCircle):
     return np.sqrt(4 * AreaCircle / np.pi)
 
 ######################### Hydraulics #########################
-RATIO_VC_ORIFICE = 0.62
+RATIO_VC_ORIFICE = con.RATIO_VC_ORIFICE
```
Comment (ethan92429) [inline aide_design/physchem.py]: replace instances of RATIO_VC_ORIFIC with con.RATIO_VC_ORIFICE at all locations in this file and take out this line.

File: aide_design/unit_process_design/prefab/lfom_prefab_functional.py [hunk 15]
```diff
@@ -31,8 +31,8 @@
 from aide_design import utility as ut
 
 # import export inputs and define the VC coefficient
-from aide_design import expert_inputs as exp
-ratio_VC_orifice= exp.RATIO_VC_ORIFICE
+from aide_design import constants as con
+ratio_VC_orifice= con.RATIO_VC_ORIFICE
```
Comment (ethan92429) [inline aide_design/unit_process_design/prefab/lfom_prefab_functional.py]: See above.

**Comment by fletchapin** [comment]
@ethan92429 we just responded to the comments that you left on the merge request. We also went ahead and redid the file structure, getting rid of the prefab folder entirely

**Pull request merged**